src/test/java/HttpServer/ResponseWriterTest.java

```diff
@@ -33,4 +33,34 @@ public void itIsNotFoundForFaviconCurl() throws IOException {
         assert(client.received(expectation));
     }
 
+    @Test
+    public void teaIsOkay() throws IOException {
+        MockTraffic traffic = new MockTraffic();
+        MockClient client = new MockClient();
+        traffic.emulateGettingTea();
+        RequestParser parser = new RequestParser(traffic);
+        ResponseWriter writer = new ResponseWriter(parser, client);
+        parser.read();
+        writer.write();
+
+        String expectation = "HTTP/1.1 200 OK";
```

Comment:
I would expect something more like `assertEqual(response.getStatus(), 200)`.

Reply:
I looked at the current state of `ResponseWriterTest`. The idea is, if I'm testing that it *writes* responses then I should be looking at strings rather than the response domain object. However, this doesn't look good to me and I don't know what would be better: https://github.com/mathpunk/HttpServer/blob/master/src/test/java/HttpServer/response/ResponseWriterTest.java#L67